shared/output/cloud/showcase/lib/google/showcase/v1beta1/echo/client.rb

```diff
@@ -416,7 +420,6 @@ def wait request, options = nil, &block
                                    retry_policy: @config.retry_policy
 
             wrap_gax_operation = ->(response) { Gapic::Operation.new response, @operations_client }
-
```

Comment:
FWIW I preferred the line break between the lamba(s) and the rpc call, but as long as it's consistent you can make the call.

Reply:
This may change soon.